docs/bigcommerce-for-wordpress/getting-started/supported-features.md

```diff
@@ -5,74 +5,73 @@
-* Gutenberg and Classic Editor
+* Gutenberg and Classic Editor integration
 * Google accelerated mobile pages 
 * Google address autocomplete 
 * Facebook Pixel 
 * Enterprise systems integrations (connect to existing ERP suites, PIM software, OMS solutions, POS systems, or marketing automation tools)
 * Faceted search with FacetWP and search with SearchWP
 * Multi-site selling
-* Multi-channel selling (eBay, Amazon, Facebook, Instagram, Google Shopping, Square and more) and Central Management (bulk listing, automatic inventory syncing, unified order and fulfillment management)
+* Multi-channel selling (eBay, Amazon, Facebook, Instagram, Google Shopping, Square, and more) and Central Management (bulk listing, automatic inventory syncing, unified order and fulfillment management)
 * Shopper wishlists
+* Shipping calculator
+* Abandoned cart recovery
+* Theme styling within customizer
+* Coupon codes
+
+## Unsupported features
+
+BC4WP does not support the following features: 
+
+* Product file upload field
+* Warranty, availability fields
+* Persistent cart
+* Cart preview
+* Cart page promotional banners
+* Real-time syncing with BigCommerce catalog updates
+* Google Analytics Enhanced Ecommerce
 
 ### Supported payment gateways
 
-BigCommerce has over 65 payment gateway integrations available out-of-the box, serving 100+ countries and over 250 local payment methods. BigCommerce for WordPress currently supports the following payment gateways:
+BigCommerce has over 65 payment gateway integrations available out-of-the box, serving 100+ countries and over 250 local payment methods. BC4WP supports the following payment gateways:
 
 * PayPal Express (now called PayPal Checkout)
 * PayPal through Braintree
-* All non-hosted (API) gateways that are supported by Optimized One Page Checkout 
+* All non-hosted (API) gateways that are supported by Optimized One-Page Checkout 
 
 ### Unsupported payment gateways
 
-The following payment gateways are currently not supported:
+BC4WP does not support the following payment gateways:
 
 * Amazon Pay
 * AfterPay
 * Google Pay
 * Masterpass
 
-
-## BigCommerce for WordPress unsupported features
-
-The BigCommerce for WordPress plugin does not currently support the following features: 
-
-* Product file upload field
-* Warranty, availability fields
-* Persistent cart
-* Cart preview
-* Cart page promotional banners
-* Real-time syncing with BigCommerce catalog updates
-* Google Analytics Enhanced Ecommerce
-
 ## General product roadmap
 
-We plan on supporting and improving the following features in the BigCommerce for WordPress plugin:
+We plan on supporting and improving the following features in the BC4WP plugin:
 
 * Product modifier improvements
 * Product variant images
 * Moderation and management of product reviews
-* Additional theme styling within customizer
 * Multi-currency
-* Abandoned cart saver 
-* Shipping calculation on cart page
 * Promotional banner support
 * Page builder plugins support

```

Comment:
```suggestion
* Page Builder plugins support

```
Should Page Builder be capitalized?

Reply:
This whole section will likely be removed soon, but in this case, I would not capitalize page builder. This is referring to third-party page builder plugins, of which there are many (Elementor, Divi, Visual Composer). This doesn't specify which page builders they plan to support, just that there might be integrations with some of them in the future.